### PR TITLE
CONSOLE-3081: [Dark theme] Changes to horizontal-nav to enable dark theme support

### PR DESCRIPTION
**Pull request by sg00dwin**
Horizontal nav with dark theme and light theme

Screen Shot 2022-03-04 at 12 31 35 PM
Screen Shot 2022-03-04 at 12 30 58 PM

### CODE REVIEW

File: frontend/public/components/_horizontal-nav.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 $co-m-horizontal-nav__menu-item-link-padding-lr: $pf-global-gutter;
 
 .co-m-horizontal-nav__menu {
-  border-bottom: 1px solid $color-grey-background-border;
+  border-bottom: 1px solid var(--pf-global--BorderColor--100);
```
Comment (jhadvig) [inline frontend/public/components/_horizontal-nav.scss:4]: wondering if there is a list of css vars that need to be switch …eg:
`$color-grey-background-border`  - for -> `var(--pf-global--BorderColor--100)` ?

Comment (sg00dwin) [reply frontend/public/components/_horizontal-nav.scss:4]: I've included a find and replace for `$color-grey-background-border` in this PR https://github.com/openshift/console/pull/11136

**Review by jerolimov — COMMENTED** [review]
Tested the detail page in dev perspective, looks as expected and as shown on your screenshot.

Topology sidebar navigation is also updated and fine:

image

image

LGTM from dev perspective!

**Comment by vikram-raj** [comment]
Propagate from epic:
/label docs-approved
/label px-approved

**Comment by sg00dwin** [comment]
QE Approver:
/assign @yanpzhan

**Comment by yanpzhan** [comment]
Checked on admin console with dark theme, the horizontal nav on related pages works as expected 
/label qe-approved

**Pull request merged**